src/uct/ib/base/ib_verbs.h

```diff
@@ -22,71 +22,16 @@
 #include <ucs/type/status.h>
 #include <ucs/debug/log.h>
 
-#if HAVE_VERBS_ECE
-typedef struct ibv_ece uct_ibv_ece_t;
-#else
-typedef struct {
-    uint32_t options;
-} uct_ibv_ece_t;
-#endif
+#ifndef HAVE_DECL_IBV_ECE
 
-static inline
-int uct_ib_query_ece(struct ibv_qp *qp, uct_ibv_ece_t *ece) {
-#if HAVE_VERBS_ECE
-    return ibv_query_ece(qp, ece);
-#else
-    return UCS_ERR_UNSUPPORTED;
-#endif
-}
-static inline
-int uct_ib_set_ece(struct ibv_qp *qp, uct_ibv_ece_t *ece) {
-#if HAVE_VERBS_ECE
-    return ibv_set_ece(qp, ece);
-#else
-    return UCS_ERR_UNSUPPORTED;
-#endif
-}
+struct ibv_ece {
+    uint32_t options;
```

Comment:
maybe it's no longer needed now that we have the wrapper function uct_ib_set_ece

Reply:
@yosefe The prototype of ```uct_ib_set_ece``` is ```ucs_status_t uct_ib_set_ece(uct_ib_device_t *dev, struct ibv_qp *qp, uint32_t ece_val)```
If we don't define the struct here, then it needs to use ```#ifdef HAVE_DECL_IBV_ECE....``` in ```uct_ib_set_ece```.

Could we still keep the struct here?

Reply:
IMO better move ifdef inside set_ece and check_ece if these are the only 2 places that need it

Reply:
Done, removed.